saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/provider/service/registration/RelyingPartyRegistration.java

```diff
@@ -177,6 +188,51 @@ public Saml2MessageBinding getAssertionConsumerServiceBinding() {
 		return this.assertionConsumerServiceBinding;
 	}
 
+	/**
+	 * Get the <a href=
+	 * "https://wiki.shibboleth.net/confluence/display/CONCEPT/MetadataForIdP#MetadataForIdP-Logout">SingleLogoutService</a>
```

Comment:
Since this is pattern extends beyond the SAML SLO support, I've added https://github.com/spring-projects/spring-security/issues/9510 to address this and the related comments.